projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,25 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: monitoring.coreos.com/v1alpha1
```

Comment:
why are you using alpha instead of just v1?

Reply:
That's whats given in the docs - 
AlertmanagerConfig [monitoring.coreos.com/v1alpha1]
https://docs.openshift.com/container-platform/4.10/rest_api/monitoring_apis/alertmanagerconfig-monitoring-coreos-com-v1alpha1.html#specification